src/main/java/uk/gov/hmcts/reform/coh/domain/OnlineHearing.java

```diff
@@ -44,16 +45,20 @@
     @Column(name = "owner_reference_id")
     private String ownerReferenceId;
 
-    @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "online_hearing_state_id", nullable = false)
+    @OneToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "online_hearing_state_id")
+    @JsonProperty("current_online_hearing_state")
```

Comment:
Json annotations can be safely deleted from this class because the classes which are responsible for handling requests & responses for online hearings are defined in controller/onlinehearing/